s3proxy/deploy/create_cert.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+# service is the name of the s3proxy service in kubernetes.
```

Comment:
```suggestion
# $service is the name of the s3proxy service in kubernetes.
```
I think this makes it faster to parse out the variable. I don't know of any style guides for bash commenting tough.

Reply:
Replaced by cert-manager solution.